### ISSUE TITLE
Disable Debug logs for Dependency linker 

### ISSUE THREAD

**Issue by rakshitcgupta** [openzipkin/zipkin-dependencies#177]
Hi, How can I switch off Debug logs for 

20/08/13 07:46:39 DEBUG DependencyLinker: cannot find remote ancestor; skipping
--
  | 20/08/13 07:46:39 DEBUG DependencyLinker: building trace tree: traceId=74187998103a1756
  | 20/08/13 07:46:39 DEBUG DependencyLinker: traversing trace tree, breadth-first
  | 20/08/13 07:46:39 DEBUG DependencyLinker: processing {"traceId":"74187998103a1756","id":"74187998103a1756","kind":"SERVER","name":"post

Basically I need to switch off all debug logs for Production go live

**Comment by rakshitcgupta** [comment]
Hi, the problem was solved adding log4j.logger.zipkin2.internal.DependencyLinker=INFO to log4j properties
